daemon/lokinet-bootstrap.cpp

```diff
@@ -28,6 +29,24 @@ namespace
     std::cout << msg << std::endl;
     return 1;
   }
+
+  int
+  print_help(std::string exe)
+  {
+    std::cout << R"(Lokinet bootstrap.signed fetchy program thing
+
+Downloads the initial bootstrap.signed for lokinet into a local file from a default or user defined server via the reachable network.
+
+Usage: )" << exe
+              << R"( [bootstrap_url [output_file]]
+
+bootstrap_url can be a named value in which we will use a predefined url that
```

Comment:
can't hurt to enumerate the list of options here

Reply:
Huh.  I'm pretty sure there *was* an enumeration of the values here when I looked at it yesterday.